src/reject.rs

```diff
@@ -509,6 +543,11 @@ unit_error! {
     pub UnsupportedMediaType: "The request's content-type is not supported"
 }
 
+unit_error! {
```

Comment:
This is probably not a `unit_error` if the `scheme` is going to be provided

Reply:
what other type of error is it then? I just tried make it return Unauthorized request, I am not a full pro of warps internals ;)

Reply:
[MissingHeader](https://github.com/seanmonstar/warp/pull/751/files/7b357e450c292dc252d40325d2aa82c56994a3e9#diff-3b90cd596f94fb9464f356d926c42bb98767b769f938613f9f7e971d3e328c3cR553) would be an error you could model this off of.

Reply:
thx, will look into it